app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -0,0 +1,165 @@
+        # ...and 5 am to 5:45 am ET on 2nd Sunday of month.
+        { cron: '0 5 * * Sun#2', duration: 45 },
+      ],
+      'NC' => [
+        # Daily, Midnight to 7:00 am. ET.
+        { cron: '0 0 * * *', duration: 7 * 60 },
+        # Sundays from 5am. till Noon
+        { cron: '0 5 * * Sun', duration: 7 * 60 },
+      ],
+      # NM: "Sunday mornings." (not modeling; too vague)
+      'NY' => [
+        # Sunday maintenance 8 pm. to 9 pm. ET.
+        { cron: '0 20 * * Sun', duration: 60 },
+      ],
+      'PA' => [
+        # Sunday maintenance may occur, often between 5:30 am. & 7:00 am. ET
+        { cron: '30 5 * * Sun', duration: 90 },
+      ],
+      'SC' => [
+        # Sunday maintenance from 7:00 pm. to 10:00 pm. ET.
+        { cron: '0 19 * * Sun', duration: 3 * 60 },
+      ],
+      'TX' => [
+        # Downtime on weekends between 9 pm ET to 7 am ET.
+        { cron: '0 21 * * Sat,Sun', duration: 10 * 60 },
+      ],
+      'VA' => [
+        # Sunday morning maintenance 3:00 am. to 5 am. ET.
+        { cron: '0 3 * * Sun', duration: 120 },
+        # Daily maintenance from 5 am. to 5:30 am.
+        { cron: '0 5 * * *', duration: 30 },
+        # "Might not respond for short spells, daily between 7 pm  and 8:30 pm." (not modeling this)
+      ],
+      'VT' => [
+        # Daily maintenance from midnight to 5 am. ET.
+        { cron: '0 0 * * *', duration: 5 * 60 },
+      ],
+      'WA' => [
+        # Maintenance from Saturday 9:45 pm. to Sunday 8:15 am. ET.
+        { cron: '45 21 * * Sat', duration: 10.5 * 60 },
+      ],
+      'WI' => [
+        # Downtime on Tuesday – Saturday typically between 3 – 4 am ET.
+        { cron: '0 3 * * Tue-Sat', duration: 60 },
+        # Downtime on Sunday from 6 – 10 am. ET.
+        { cron: '0 6 * * Sun', duration: 4 * 60 },
+      ],
+      'WV' => [
+        # Occasional Sunday maintenance from 6:00 am. to noon ET.
+        { cron: '0 6 * * Sun', duration: 6 * 60 },
+      ],
+      'WY' => [
+        # Daily, 2 am. to 5 am. ET.
+        { cron: '0 2 * * *', duration: 3 * 60 },
+      ],
+    }.freeze
+
+    class << self
+      # _All_ AAMVA maintenance windows are expressed in 'ET' (LG-14028)
+      TZ = 'America/New_York'
+
+      def in_maintenance_window?(state)
+        return false unless (windows = windows_for_state(state))
+
+        Time.use_zone(TZ) do
+          windows.any? { |window| window.cover?(Time.zone.now) }
+        end
+      end
+
+      # private
+
+      def windows_for_state(state)
+        return [] unless MAINTENANCE_WINDOWS[state]
+
+        Time.use_zone(TZ) do
+          MAINTENANCE_WINDOWS[state].collect do |window|
+            cron = Fugit.parse_cron(window[:cron])
+            prev_instance = cron.previous_time.to_t
+            # We don't need to consider the _next_ window, because it is in the future.
+            # And we're never in the future yet.
+            # next_instance = cron.next_time.to_t
```

Comment:
This was sort of an internal monologue and will be removed, but I wanted to leave it for conversation initially.

I first wrote this to check `.next_time` and `.previous_time`, but I eventually realized that `next_time` was necessarily always in the future.